src/verifiers/documentStatus/didSignedDocumentStatus/didSignedDocumentStatus.test.ts

```diff
@@ -0,0 +1,33 @@
+import { OpenAttestationDidSignedDocumentStatus } from "./didSignedDocumentStatus";
+
+describe("skip", () => {
+  it("should return skip message", () => {});
+});
+
+describe("test", () => {
+  describe("v2", () => {
+    it("should return false for documents not signed by DID", () => {});
+    it("should return true for documents where any issuer is using the `DID` identity proof", () => {});
+    it("should return true for documents where any issuer is using the `DNS-DID` identity proof", () => {});
+  });
+  describe("v3", () => {
```

Comment:
I would suggest to keep tests for v3 skipped (but keep the description) we will implement once v3 is ready

Reply:
ok